### PR TITLE
Updated default monitor frequency to 1

### PR DESCRIPTION
**Pull request by kthui**
The default monitor frequency is changed to 1, to reduce the load of cpu-monitor on the processor. It is because higher load on the processor might affect the result of the perf_analyzer.

### CODE REVIEW

**Review by Tabrizian — APPROVED** [review]

**Comment by kthui** [comment]
Waiting for CI. Will merge after it is completed.

**Pull request merged**
